sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestData.scala

```diff
@@ -73,6 +74,17 @@ private[sql] trait SQLTestData { self =>
     df
   }
 
+  protected lazy val timestampData: DataFrame = {
+    val df = spark.sparkContext.parallelize(
+      TestDataTimestamp(new Timestamp(1420140300000L)) :: // 2015-01-01 20:25:00
+        TestDataTimestamp(new Timestamp(1320140300000L)) :: // 2011-11-01 10:38:20
+        TestDataTimestamp(new Timestamp(1520140300000L)) :: // 2018-03-04 06:11:40
+        TestDataTimestamp(new Timestamp(-1409632500000L)) :: // 1925-05-01 19:44:32
```

Comment:
Do you set the fractional part of seconds to zeros intentionally?

Reply:
No, this wasn't intentional. I can add some fractions if you like.